src/modules/mc_pos_control/multicopter_nudging_params.c

```diff
@@ -32,34 +32,34 @@
  ****************************************************************************/
 
 /**
- * Enable user assisted descent for autonomous land routine
+ * Enable nudging based on user input during autonomous land routine
  *
- * When enabled, descent speed will be:
+ * Using stick input the vehicle can be moved horizontally and yawed.
+ * The descend speed is amended:
  * stick full up - 0
  * stick centered - MPC_LAND_SPEED
  * stick full down - 2 * MPC_LAND_SPEED
  *
- * Additionally, the vehicle can be yawed and moved laterally using the other sticks.
  * Manual override during auto modes has to be disabled to use this feature (see COM_RC_OVERRIDE).
  *
  * @min 0
  * @max 1
- * @value 0 Fixed descent speed of MPC_LAND_SPEED
- * @value 1 User assisted descent speed
+ * @value 0 Nudging disabled
+ * @value 1 Nudging enabled
  * @group Multicopter Position Control
  */
 PARAM_DEFINE_INT32(MPC_LAND_RC_HELP, 0);
```

Comment:
I planned to change names and groups in the next step since nudging is only for autonomous flight and would not be in the group for e.g. position mode and then I'd probably also change the prefix since it has nothing to do anymore with position control...